src/deprecations.jl

```diff
@@ -394,5 +394,8 @@ function SubdivisionOfPoints{T}(obj::Polymake.BigObject) where T<:scalar_types
   return SubdivisionOfPoints{T}(obj, _detect_default_field(T, obj))
 end
 
+@deprecate is_skewsymmetric_matrix(B::MatElem{T}) where T <: RingElem is_alternating(B)
+@alias isskewsymmetric_matrix is_skewsymmetric_matrix
+
```

Comment:
Do we really want this deprecation? I find it very misleading from its name. As the next release will be breaking anyway, we could skip adding it.